src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -195,6 +196,43 @@
 				:questions="frequentlyAskedQuestions"
 			/>
 		</div>
+
+		<!-- Monthly Good False Door Exp Optional Choice -->
+		<kv-lightbox
+			:visible="showActiveLightbox"
+			@lightbox-closed="closeLightbox"
+		>
+			<h2 class="tw-text-h2 tw-text-center">
```

Comment:
design wants this centered and not in line with the close icon, looks like putting this in `title` or `<template #header>` as a custom header keeps this in the same row as the close icon